request/request.js

```diff
@@ -31,7 +31,7 @@ module.exports = function($window, Promise) {
 	function normalize(args, extra) {
 		if (typeof args === "string") {
 			var url = args
-			args = extra || {}
+			args = extra ? Object.assign(extra, {}} : {}
```

Comment:
We still support IE, which doesn't have `Object.assign`.

Reply:
what do you suggest ?
JSON.parse/stringify ?

Reply:
@nraibaud Ignore the comment. I hit a wrong button.

Reply:
ok i will add some tests so

Reply:
```js
args = {}
if(extra) for(var prop in extra)
  args[prop] = extra[prop]
```
?